doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -43,7 +43,174 @@ All the JSON log types share a common structure:
 ::
 
 
-  {"timestamp":"2009-11-24T21:27:09.534255","event_type":"TYPE", ...tuple... ,"TYPE":{ ... type specific content ... }}
+  {"timestamp":"2009-11-24T21:27:09.534255","flow_id":ID_NUMBER, "event_type":"TYPE", ...tuple... ,"TYPE":{ ... type specific content ... }}
+
+Flow id
```

Comment:
Should this be `Flow ID` or `flow_id`?

Reply:
Important question. Looking through some other cases, I see that we have `ftp_data` as `FTP_DATA` , so I'd say it should be `flow_id`. Thanks!